core/src/io/github/teamfractal/RoboticonQuest.java

```diff
@@ -1,4 +1,7 @@
-
+/*  JBT Assessment 4 Page: http://robins.tech/jbt/assfour.html
+ *  JBT Changes to this file:
+ *  Removed partial patch for a bug, which was fixed elsewhere
```

Comment:
This is quite hard to understand what you mean. From someone who doesnt know what the bug is or where the elsewhere is as they wont be looking at it in a PR like us